### PR TITLE
Update naming and links

### PR DESCRIPTION
**Pull request by 0xalank**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,26 +13,26 @@ to them very soon! More below.
...
+Pelagus is currently being built by a core team in collaboration with a community
+on the [Pelagus Community Discord server](https://chat.pelaguswallet.io). **Discord is
 the right place to start discussions on new features and bugs.** The community
 on Discord, led by a few designated folks will help to funnel these into
 well-organized GitHub issues for features and bugs, as well as organize folks
 to tackle any issues they’re interested in. For the time being, the core team
 will be charged with reviewing, critiquing, and ultimately merging new work.
 
-⭐️ Check out our ["good first issues" tag](https://github.com/tahowallet/extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) for existing bugs that are more approachable.
+⭐️ Check out our ["good first issues" tag](https://github.com/pelaguswallet/extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) for existing bugs that are more approachable.
```
Comment (wizeguyy) [inline CONTRIBUTING.md]: This link is broken

File: ui/pages/Settings.tsx [hunk 2]
```diff
@@ -36,15 +36,15 @@ const FAQ_URL =
 const FOOTER_ACTIONS = [
   {
     icon: "icons/m/discord",
-    linkTo: "https://chat.pelaguswallet.io/",
+    linkTo: "https://discord.gg/pcaA5EapZk",
```
Comment (wizeguyy) [inline ui/pages/Settings.tsx:39]: please squash this commit into the original commit which attempted to rename these.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -13,26 +13,26 @@ to them very soon! More below.
 
 ## Deciding What to Work On
 
-Taho is currently being built by a core team in collaboration with a community
-on the [Taho Community Discord server](https://chat.taho.xyz). **Discord is
+Pelagus is currently being built by a core team in collaboration with a community
+on the [Pelagus Community Discord server](https://chat.pelaguswallet.io). **Discord is
```
Comment (wizeguyy) [inline CONTRIBUTING.md:17]: Another broken link

Comment (0xalank) [reply CONTRIBUTING.md:17]: fixed

**Pull request closed without merging**